vpr/src/device/rr_graph_view.h

```diff
@@ -0,0 +1,99 @@
+#ifndef _RR_GRAPH_OVERLAY_
+#define _RR_GRAPH_OVERLAY_
+
+#include <exception>
+
+#include "rr_graph_fwd.h"
```

Comment:
(like in the other place, keep the headers sorted. Since this is a new file, we can start right away using best practices here)

Reply:
Agree